tests/pytest.ini

```
@@ -1,7 +1,9 @@
 [pytest]
 ; logging options
-log_cli = 1
-log_level = WARNING
+log_cli = False
+addopts = -v --nf --tb=short
```

I find long-form options to be more self documenting.

```suggestion
addopts = --verbose --new-first --tb=short
```

---

```
--nf, --new-first     run tests from new files first, then the rest of the tests sorted by file mtime
```
This seems like a thing we wouldn't want in CI.  I would probably choose regular order in CI for now.  If we want to mix it up then truly random order instead of whatever-git-does would likely be better.  It looks like you can use a [`PYTEST_ADDOPTS` environment variable](https://docs.pytest.org/en/6.2.x/example/simple.html) to get this for yourself locally.

---

Not sure I like the short traceback idea, but I'm willing to give it a try.  If anything the long form ought to make it easier to find the failure in the log.  We'll see how it looks thought.